vendors/materialize/css/materialize.css

```diff
@@ -5582,6 +5582,7 @@ button.btn-floating {
   background-color: #e4e4e4;
   margin-bottom: 5px;
   margin-right: 5px;
+  width: max-content;
```

Comment:
It is not a good idea change the vendor files since these are from a third party and used across the project. Custom CSS goes here https://github.com/coala/coalaCSS

Reply:
i have a little query that should i close this pull request as vendor files could not be changed.
and make a pull request in coalaCSS repository after modification ?

Reply:
Yes, absolutely. It'll be really great if you can open the MR in coalaCSS since modifying vendor files directly makes it problematic to update them.